crm-project/src/main/java/CRM/utils/Validations.java

```diff
@@ -209,4 +232,36 @@ public static <T> void setContentToFieldIfFieldExists(T object, String fieldName
     public static void throwAttributeAlreadyExistsForBoard(Attribute attribute, String className){
         throw new NonUniqueObjectException(ExceptionMessage.ATTRIBUTE_ALREADY_IN_DB.toString(), attribute.getId(), className);
     }
+
+    public static boolean checkIfFieldIsCustomObject(UpdateField fieldName){
+        return fieldName.equals(STATUS) || fieldName.equals(TYPE) || fieldName.equals(PARENT_ITEM);
+    }
+
+    public static boolean checkIfFieldIsNonPrimitive(UpdateField fieldName) {
+        return fieldName.equals(DUE_DATE);
+    }
+
+    public static void checkIfParentItemIsNotTheSameItem(UpdateField fieldName, Long itemId, Long parentItemId) {
+        if(fieldName.equals(PARENT_ITEM) && Objects.equals(itemId, parentItemId)){
+            throw new IllegalArgumentException(ExceptionMessage.PARENT_ITEM_ERROR.toString());
+        }
+    }
+
+    public static Class getFieldObjectRepository(UpdateField fieldName){
+        Class repo = null;
+        switch (fieldName) {
+            case STATUS:
+                repo = StatusRepository.class;
+                break;
+            case TYPE:
+                repo = TypeRepository.class;
+                break;
+            case PARENT_ITEM:
+                repo = ItemRepository.class;
+                break;
+            default:
+                break;
+        }
+        return repo;
+    }
```

Comment:
That looks like a perfect function for all of our classes. Why not put it in "common" section?
I think you can extend it and put more types there so we can call it for other functions as well.